server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTestHelper.java

```diff
@@ -0,0 +1,123 @@
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+* The StorageHandlerTestHelper class holds functions that are used in multiple database test files.
+*/
+public class StorageHandlerTestHelper {
+  static DatabaseClient dbClient;
+  private static final String INSTANCE_ID = "coffeehouse-instance-test";
+  private static final String DATABASE_ID = "coffeehouse-db-test";
+
+  public static DatabaseClient setUpHelper() {
+    // Create spanner service and database client
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+    DatabaseId db = DatabaseId.of(options.getProjectId(), INSTANCE_ID, DATABASE_ID);
+    dbClient = spanner.getDatabaseClient(db);
+    return dbClient;
+  }
+
+  public static void setUpClearDb() {
+    // Delete all the data while keeping the existing database
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(Mutation.delete("Persons", KeySet.all()));
+    mutations.add(Mutation.delete("Books", KeySet.all()));
+    mutations.add(Mutation.delete("Clubs", KeySet.all()));
+    mutations.add(Mutation.delete("Memberships", KeySet.all()));
+    dbClient.write(mutations);
+  }
+
+  public static void insertPerson() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertBuilder("Persons")
+        .set("userId")
+        .to("person")
+        .set("email")
+        .to("person@test.com")
+        .set("nickname")
+        .to("person")
+        .set("pronouns")
+        .to("she/he/they")
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public static void insertClub() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Clubs")
+        .set("clubId")
+        .to("club")
+        .set("bookId")
+        .to("book")
+        .set("description")
+        .to("A club")
+        .set("name")
+        .to("Club")
+        .set("ownerId")
+        .to("owner")
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public static void insertMembership() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertBuilder("Memberships")
+        .set("userId")
+        .to("person")
+        .set("clubId")
+        .to("club")
+        .set("membershipType")
+        .to(MembershipConstants.MEMBER)
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build());
+    dbClient.write(mutations);
+  }
+
+  public static void insertBook() {
+    List<Mutation> mutations = new ArrayList<>();
+    mutations.add(
+      Mutation.newInsertOrUpdateBuilder("Books")
+        .set("bookId")
+        .to("book")
+        .set("author")
+        .to("book")
+        .set("isbn")
+        .to("97800609459470")
+        .set("title")
+        .to("book")
+        .build());
+    dbClient.write(mutations);
+  }
+}
```

Comment:
```suggestion
}

```
Newline at end of file.